app/controllers/application_controller.rb

```diff
@@ -315,12 +315,18 @@ def show_progress?
     false
   end
 
-  def show_offseason_filing_banner?
-    return false if hub?
-
-    app_time >= Rails.configuration.tax_deadline && app_time <= Rails.configuration.end_of_in_progress_intake
+  def homepage_banner
+    if app_time <= Rails.configuration.tax_deadline
+      :before_tax_deadline
+    elsif app_time.between?(Rails.configuration.tax_deadline, Rails.configuration.end_of_in_progress_intake) #&& open_for_gyr_intake?
```

Comment:
unclear maybe should used open_for_gyr_intake? ; this wouldn't impact us till tax deadline and we have yet to set `config.tax_deadline` for next year anyways